docs/en/sql-reference/aggregate-functions/reference/aggthrow.md

```diff
@@ -0,0 +1,37 @@
+---
+slug: /en/sql-reference/aggregate-functions/reference/aggthrow
+sidebar_position: 101
+---
+
+# aggThrow
+
+This function can be used for the purpose of testing exception safety. It will throw an exception on creation with the specified probability.
+
+**Syntax**
+
+```sql
+aggThrow(throw_prob)
+```
+
+**Arguments**
+
+- `throw_prob` — Probability to throw on creation. [Float64](../../data-types/float.md).
```

Comment:
Looking at the example (l. 29) and l. 18, the range of the probability, e.g. `[0.0 - 1.0)` or `[0.0 - 100.0)`, is not clear.

Reply:
Just in case, if we have percentages anywhere, we should fix and change to ratios.